nomad/server.go

```diff
@@ -1450,6 +1454,27 @@ func (s *Server) Stats() map[string]map[string]string {
 	return stats
 }
 
+// EmitRaftStats is used to export metrics about raft indexes and state store snapshot index
+func (s *Server) EmitRaftStats(period time.Duration, stopCh <-chan struct{}) {
+	for {
+		select {
+		case <-time.After(period):
```

Comment:
very nit-picky - `timer.Ticker` is slightly nicer for using in loops, to reuse channels and timer mechanisms.